WordPress/gutenberg
# Inserter: Replace the selected block if empty

closes #3627

this PR update the inserter to replace the selected block if it's an empty block.

**Testing instructions**

- Add an empty paragraph block
- Select it
- Add an image block from the inserter
- The image block should replace the empty block instead of adding a new block
